README.md

```diff
@@ -94,142 +88,150 @@ Class | Method | HTTP request | Description
- - [FacternApi.SettleAccountRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/SettleAccountRequest.md)
- - [FacternApi.Statement](https://github.com/Factern/factern-client-javascript/blob/master/docs/Statement.md)
- - [FacternApi.Template](https://github.com/Factern/factern-client-javascript/blob/master/docs/Template.md)
- - [FacternApi.UpdateApplicationRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/UpdateApplicationRequest.md)
- - [FacternApi.UpdateApplicationResponse](https://github.com/Factern/factern-client-javascript/blob/master/docs/UpdateApplicationResponse.md)
- - [FacternApi.Watch](https://github.com/Factern/factern-client-javascript/blob/master/docs/Watch.md)
- - [FacternApi.WatchEvent](https://github.com/Factern/factern-client-javascript/blob/master/docs/WatchEvent.md)
- - [FacternApi.WriteRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/WriteRequest.md)
- - [FacternApi.CreateInformationRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateInformationRequest.md)
- - [FacternApi.CreateMemberRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateMemberRequest.md)
- - [FacternApi.CreateNamedRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateNamedRequest.md)
- - [FacternApi.UpdateStatusRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/UpdateStatusRequest.md)
- - [FacternApi.CreateApplicationRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateApplicationRequest.md)
- - [FacternApi.CreateDomainRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateDomainRequest.md)
- - [FacternApi.CreateFieldRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateFieldRequest.md)
- - [FacternApi.CreateFilterRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateFilterRequest.md)
- - [FacternApi.CreateGroupRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateGroupRequest.md)
- - [FacternApi.CreateInDomainRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateInDomainRequest.md)
- - [FacternApi.CreateInterfaceRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateInterfaceRequest.md)
- - [FacternApi.CreateLabelListRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateLabelListRequest.md)
- - [FacternApi.CreateScopeRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateScopeRequest.md)
- - [FacternApi.CreateTemplateRequest](https://github.com/Factern/factern-client-javascript/blob/master/docs/CreateTemplateRequest.md)
+ - [FacternApi.Account](docs/Account.md)
```

Comment:
Sorry, I wasn't clear when I only pointed out the one section before...  All relative url's need to be full paths or they won't work when viewing on npm